lib/aspose-imaging-cloud/api_client.rb

```diff
@@ -130,7 +130,7 @@ def build_request(http_method, path, opts = {})
       conn = Faraday.new url, { :params => query_params, :headers => header_params } do |f|
         f.request :multipart
         f.request :url_encoded
-        f.adapter :net_http_persistent, pool_size: 5
+        f.adapter Faraday.default_adapter
```

Comment:
@dmitrypasyutin This change was caused by a wrong template for codegen, could you please exclude it from the PR? I will fix the template